runelite-client/src/main/java/net/runelite/client/plugins/timers/TimersPlugin.java

```diff
@@ -159,8 +159,8 @@
 	private int lastAnimation;
 	private boolean widgetHiddenChangedOnPvpWorld;
 	private ElapsedTimer tzhaarTimer;
-	private int imbuedHeartClickTick = -1;
-	private int lastBoostedMagicLevel = -1;
+	private TimerTimer imbuedHeartTimer;
```

Comment:
This doesn't need to be a field, it's never referenced anywhere except to null it.

Reply:
There's a relevant check on line 317. Otherwise we would re-create the timer every 60 game ticks when the server sends an update on the varbit.
However I agree this should be changed to a flag, I saved a Timer reference since my first idea would be to update the time on varbit changed, which I later forgo.

Reply:
Actually, scratch my above comment. I simply add the timer once we get the initial cooldown value (70) and remove it on 0 and this can be removed. I don't know why I didn't think of it before :|